emulators/xhyve/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        mist64 xhyve 0.2.0 v
-revision            2
+github.setup        mist64 xhyve 1f1dbe3 
+revision            0
```

Comment:
You may delete the whole line setting the revision to zero as that's the default anyway.

But you have to add an explicit version, else there will be troubles with incrementing the version number. You may use something like
```
version 0.2.1-git-1f1dbe3
```
(just making things up).